src/components/views/messages/MessageActionBar.tsx

```diff
@@ -335,9 +335,14 @@ export default class MessageActionBar extends React.PureComponent<IMessageAction
                     'mx_MessageActionBar_expandMessageButton': !this.props.isQuoteExpanded,
                     'mx_MessageActionBar_collapseMessageButton': this.props.isQuoteExpanded,
                 });
+                const shiftClickText = `| ${_t("⇧+click")}`;
                 toolbarOpts.push(<RovingAccessibleTooltipButton
                     className={expandClassName}
-                    title={this.props.isQuoteExpanded ? _t("Collapse quotes | ⇧+click") : _t("Expand quotes | ⇧+click")}
+                    title={this.props.isQuoteExpanded ? _t("Collapse quotes %(shiftClickText)s", {
```

Comment:
This was correct before - we shouldn't be breaking down the strings into parts as it can affect translations negatively. Even the formatting pipe is important to have in the translation.

Reply:
@turt2live The problem is that `|` is being handles as a one or many type in translations and will be ignored and replaced by default. Maybe we should replace it only for case of `|other` and `|one` exactly and skip in case there's actual text?
E.g. ```
    "%(count)s members|other": "%(count)s members",
    "%(count)s members|one": "%(count)s member",
    "%(count)s rooms|other": "%(count)s rooms",
    "%(count)s rooms|one": "%(count)s room",
    ```

Reply:
We probably need another way to represent keyboard shortcuts inline then. This is the first time we're doing it, which raises questions of its own.

Reply:
@turt2live we have already raised this problem internally with @janogarcia , as this way of doing it is pretty bad. I think it makes sense to merge this though and focus on making the inline shortcuts thing being useful ASAP. What do you think?
I could change the translations script too, but either this solution or updating the script both sounds like a temporary solutions, and not the correct fix (which would be making a component that renders it correctly).

Reply:
Some other options for workarounds could be using one of the line drawing characters rather than the pipe (eg. `|`) or maybe just putting it in brackets?

Reply:
Changed it to `│` and seems to look very fine. @janogarcia please give final verdict. 
<img width="355" alt="CleanShot 2021-09-21 at 12 49 30@2x" src="https://user-images.githubusercontent.com/3636685/134158098-8d461cff-2597-427f-a3e1-9b9b7b58d0bb.png">

Reply:
> Changed it to `│`

Yeah, I think it's good enough as an interim solution. 👍 That said we should follow it up (on a separate issue) by adding a proper way to inform about shortcuts in tooltips, as the current one is not optimal in many ways: localization, accessibility (screen readers), and design-wise (we can't style it separately).